src/Table/Table.js

```diff
@@ -44,7 +44,7 @@ export default class Table extends Component {
 
   static childContextTypes = { table: PropTypes.object };
 
-  getChildContext() {
+  static getChildContext() {
```

Comment:
I can't see why not. I have asked [here](https://github.com/facebook/react/issues/4864#issuecomment-252484634) just in case.